brave/browser-laptop
# Add security setting to always allow fullscreen

## Test Plan:
1. go to about:preferences#security
2. find new fullscreen select setting, hit "always allow"
3. go to https://vimeo.com, open fullscreen
4. fullscreen should be open without notification prompt 
5. Go back to fullscreen setting
6. Disable with "always ask"
7. Go to https://vimeo.com or any other video website
8. Make video fullscreen, notification should open

**QA note:**

1. When notification for fullscreen is set to "deny", even without "remember this decision" checkbox marked, option is stored and you can't undo. Bug wasn't introduced by this PR and was filled under #7010 with proper STR.

Auditors: @bsclifton, @luixxiul 

Fix #5979
